.github/workflows/Ontology_Mappings_Diagnostics.yml

```diff
@@ -17,8 +17,7 @@ jobs:
         path: ~/.cache/pypoetry/virtualenvs
         key: ${{ runner.os }}-poetry-${{ hashFiles('poetry.lock') }}
     - run: poetry --version
-    - run: poetry install --no-interaction --no-root
-      if: steps.cache-deps.outputs.cache-hit != 'true'
+    - run: poetry install
```

Comment:
@jbkoh I'm having trouble with using my installed dependencies within the GH action. Poetry shell doesn't work.